examples/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+add_executable(https_client https_client.cpp)
+
+if (WIN32)
+  target_link_libraries(https_client PRIVATE
+    boost-windows-sspi
+    )
+else()
+  target_link_libraries(https_client PRIVATE
+    Boost::headers
+    pthread
```

Comment:
The idiomatic way of enabling threading is:
```
find_package (Threads)

target_link_libraries(https_client PRIVATE Threads::Threads)
```

Reply:
Indeed. I thought I had actually done that. Will do. Thanks.